docs/fastdds/property_policies/non_consolidated_qos.rst

```diff
@@ -474,3 +474,65 @@ The different property values have the following effects on the local |DomainPar
         :start-after: <!-->TYPE_PROPAGATION_PROPERTY<-->
         :end-before: <!--><-->
         :lines: 2-4,6-17,19-20
+
+.. _property_serialize_optional_qos:
+
+Add optional QoS to EDP messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During the Endpoint Discovery Phase (EDP), |DataWriters-api| and |DataReaders-api| acknowledge each other.
+To do that, the DomainParticipants share information about their DataWriters and DataReaders with each other,
+using the communication channels established during the PDP.
+This information contains all data required to match the endpoints, such as the |Topic-api|, data type, and
+certain :ref:`QoS Policies <dds_layer_core_policy>` that might matching.
+Specific compatibility rules can be found in each QoS section of :ref:`QoS Policies <dds_layer_core_policy>`.
+
+However, there are some QoS that are not mandatory for matching, but can be useful to have in the EDP messages.
+Property ``fastdds.serialize_optional_qos`` allows the user to include these optional QoS in the EDP messages.
```

Comment:
We shoould also say that it is configured at participant level, so all associated endpoints to it will send the extra qos.